probables/blooms/countingbloom.py

```diff
@@ -62,6 +63,24 @@ def __init__(
             hash_function=hash_function,
         )
 
+    @classmethod
+    def frombytes(cls, b: ByteString, hash_function: Union[HashFuncT, None] = None) -> "CountingBloomFilter":
+        """
+        Args:
+            b (ByteString): the bytes to load as a Counting Bloom Filter
+            hash_function (function): Hashing strategy function to use `hf(key, number)`
+        Returns:
+            CountingBloomFilter: A Counting Bloom Filter object
+        """
+        blm = CountingBloomFilter(
+            est_elements=1, false_positive_rate=0.1, hash_function=hash_function
```

Comment:
It'd be nice to have the params not hardcoded.

Reply:
The hardcoded params are used to setup a "blank" element that can then be loaded and updated (so those are not actually used). I can look into making it not need to do that but it would likely result in a bit less DRY code as we would have to parse the information out, return it, build the object, etc.